tests/init_db.sh

```diff
@@ -1,37 +1,47 @@
+#!/bin/bash
+
+# Run the scrpit as follows from your terminal before starting 'cargo test':
+# . ./init_db.sh
+
+export ARANGODB_HOST="10.20.1.240:8529/"
```

Comment:
The host is not accessable in travis CI and thus breaks the CI test.

I really appreciate the idea that use env var to set arangodb related parameters. Fixing these parameters in `init_db.sh` 
harms the advantage that we can flexibly set arangodb parameters with different values in multiple local development environment and CI test.

Shall we set these envionment variables in IDE or shell for local tests purpose if you don't use arangors docker container for testing? Or write antoher shell script to set env in local development?

Reply:
If we re-set the default variables in the script to 'localhost', and the root / normal username and password to whatever is required for the travis CI test, we could also make the change to the helper function returning the root/normal user name and password the way to specified above (return the default specified as const in the Rust code if no environment variable is set).

That way, the init-script should be running even without the host being accessible (as long as travis CI can run the script, that's no issue). And it would also get the right username/password combinations in the Rust tests.

Does that sound like a plan?

Reply:
Good idea.

Would you like to merge this into a develop branch or master branch?

Reply:
Well, maybe merge into master and re-create develop to have a clean starting point? I guess finally that's up to you.